Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -321,6 +321,13 @@ static String mutationObserverNotificationString()
     page->chrome().client().handleClickForDataDetectionResult(dataDetectorElementInfo, clickPointInPluginSpace);
 }
 
+void UnifiedPDFPlugin::hideActiveDataDetectorHighlightOverlay()
+{
+    auto lastKnownMousePositionInDocumentSpace = convertDown<FloatPoint>(CoordinateSpace::Plugin, CoordinateSpace::PDFDocumentLayout, lastKnownMousePositionInView());
```

Comment:
Seems a bit weird to use the last known mouse position to hide an active highlight that we must already be tracking somewhere? If you're getting pageIndex just to satisfy my future discrete mode, there will be easier ways (by just asking about which row is visible).

Reply:
This was a little forward looking, yes, but I suppose I can fix that if needed when necessary.

Reply:
> hide an active highlight that we must already be tracking somewhere

That's right, the overlay controller already tracks the active highlight, so I'll refrain from doing this.

Reply:
Fixed in https://github.com/WebKit/WebKit/commit/5bd05389d6e9401e0d119fc1afe258157c7c136f